pinot-spi/src/main/java/org/apache/pinot/spi/env/CommonsConfigurationUtils.java

```diff
@@ -43,28 +46,16 @@
  */
 public class CommonsConfigurationUtils {
   private static final Character DEFAULT_LIST_DELIMITER = ',';
+  public static final String VERSION_HEADER_IDENTIFIER = "version";
 
-  private CommonsConfigurationUtils() {
-  }
+  // usage: default header version of all configurations.
+  // if properties configuration doesn't contain header version, it will be considered as 1
+  public static final String PROPERTIES_CONFIGURATION_HEADER_VERSION_1 = "1";
```

Comment:
I don't think we need "1" and can use `null` if there is no header from segment metadata file. 

The var name is not good as it encodes the value in the name. how about `SEGMENT_METADATA_VERSION = 1` as default it's null.
```
public static final String PROPERTIES_CONFIGURATION_HEADER_VERSION_2 = "2";
```